docs/source/data_ingestion_and_preparation/load_data.md

```diff
@@ -12,31 +12,43 @@ Once data is crawled, Sycamore will use the default data preparation code to pro
 
 To use the S3 crawler, run this command and specify your S3 bucket (and optional folder):
 
-```docker run -v crawl_data:/app/.data/.s3 -e AWS_ACCESS_KEY_ID -e AWS_SECRET_ACCESS_KEY -e AWS_SESSION_TOKEN crawler_s3 [your-bucket-name]/[your-folder-name]```
+```
+docker compose run sycamore_crawler_s3 [your-bucket-name] [optional-folder-name]
 
-You can provide your AWS keys as arguments in this command, SSO, or the other ways the CLI resolves AWS credentials.
+#example
+docker compose run sycamore_crawler_s3 aryn-public sort-benchmark/pdf
+```
+
+You can provide your AWS keys as arguments in this command, SSO, or the other ways the CLI resolves AWS credentials. For instance, you can manually add these variables:
```

Comment:
So long as we do this via compose, you don't need this part. The env vars are automatically propagated.

Reply:
Ack.